+exp/ExpTest.m

```diff
@@ -26,16 +28,22 @@
 % *Note: While the experiment is running, the position of the mouse
 % cursor will be set to emulate the wheel. Press the 'c' keyboard key
 % while experiment is running to disable/enable this feature.
-%
-% Todo: set properties and methods attributes appropriately
   
-%% properties (Exposed)
+%% properties
   
-  properties % can be chosen in GUI, in command line to improve visualization, or set by 'exp.SignalsExpTest'
+  % can be set in GUI or command line to improve visualization
+  properties
     ScreenH % handle to PTB Screen which displays visual stimuli
     LivePlotFig % handle to figure for live-plotting signals
-    SignalsExpTest % 'SignalsExpTest' object which contains info for running the *Signals* experiment
-  end
+  end 
+  
+%% properties (SetAccess = ?exp.SignalsExpTest)
+
+  % set by 'exp.SignalsExpTest'
+  properties (SetAccess = ?exp.SignalsExpTest)
+    SigExpTest % 'SignalsExpTest' object which contains info for running the *Signals* experiment
```

Comment:
At some point you should get rig of SignalsExpTest.  Either adapt SignalsExp, subclass it, or fold the code into 'ExpTest'.  I think this and 'ParamEditor' as they currently stand don't conform to OOP principals.

Reply:
yes agreed, I will do this on the next iteration of changes I make to ExpTest

Reply:
Mind making this into a GitHub issue?